samples/Sentry.Samples.Uwp/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>", Scope = "member", Target = "~M:Sentry.Samples.Uwp.Action.IgnitionStart~System.Threading.Tasks.Task")]
```

Comment:
Please amend: `<Pending>`

Why is this ignoring the warning instead of just fixing it?